rllib/agents/trainer.py

```diff
@@ -174,14 +179,14 @@
     # Number of episodes to run per evaluation period. If using multiple
     # evaluation workers, we will run at least this many episodes total.
     "evaluation_num_episodes": 10,
-    # Extra arguments to pass to evaluation workers.
     # Typical usage is to pass extra args to evaluation env creator
     # and to disable exploration by computing deterministic actions
     "evaluation_config": {
         # Example: overriding env_config, exploration, etc:
         # "env_config": {...},
         # "exploration_fraction": 0,
         # "exploration_final_eps": 0,
```

Comment:
Remove the above two lines since they are deprecated? Make sure to grep the codebase for other instances.